### ISSUE TITLE
Billboard thresholds with newrelic_one_dashboard

### ISSUE THREAD

**Issue by nickbru** [newrelic/terraform-provider-newrelic#1417]
Hi there,

Thank you for opening an issue. In order to better assist you with your issue, we kindly ask to follow the template format and instructions. Please note that we try to keep the Terraform issue tracker reserved for **bug reports** and **feature requests** only. General usage questions submitted as issues will be closed and redirected to New Relic's Explorers Hub https://discuss.newrelic.com/c/build-on-new-relic/developer-toolkit.

## Please include the following with your bug report

> :warning: **Important:** Failure to include the following, such as omitting the Terraform configuration in question, may delay resolving the issue.

- [ ] Your New Relic `provider` [configuration](#terraform-configuration-files) (sensitive details redacted)
- [ ] A list of [affected resources](#affected-resources) and/or data sources
- [ ] The [configuration](#terraform-configuration-files) of the resources and/or data sources related to the bug report (i.e. from the list mentioned above)
- [ ] Description of the [current behavior](#actual-behavior) (the bug)
- [ ] Description of the [expected behavior](#expected-behavior)
- [ ] Any related [log output](#debug-output)

### Terraform Version
```
$ terraform -v
Terraform v1.0.4
on linux_amd64
+ provider registry.terraform.io/newrelic/newrelic v2.24.1
```

### Affected Resource(s)
- `newrelic_one_dashboard`
- `widget_billboard`

### Terraform Configuration
> Please include your `provider` configuration (sensitive details redacted) as well as the configuration of the resources and/or data sources related to the bug report.
```hcl
## NR Provider.
terraform {
  required_providers {
    newrelic = {
      source  = "newrelic/newrelic"
      version = "2.24.1"
    }
  }
}

provider "newrelic" {
  # Configuration options 
  api_key    = var.api_key
  account_id = var.account_id
  region     = "US"
}
## main.tf
resource "newrelic_one_dashboard" "test-dashboard" {
  name = "Test Dashboard"

  page {
    name = "Test Dashboard"
    
  widget_billboard {
      title    = "Test Billboard"
      row      = 5
      column   = 1
      height   = 1
      width    = 1
      critical = null
      warning  = null

      nrql_query {
        query = "Query"
      }
}
```

### Actual Behavior
With "null" defined for critical & warning or critical and warning being omitted the billboard widget and dashboard page are created the threshold values are set to "0"

### Expected Behavior
If undefined or defined as "null" the value should not be defined. This is the behavior with the api.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Important Factoids
None

### References
None

**Comment by plimpton** [comment]
Getting this problem as well, not sure why there is no feedback yet

**Comment by adri** [comment]
Same problem here :)

**Comment by kidk** [comment]
There's a pull request open to resolve this issue #1528 Thanks everyone for your patience

**Comment by kidk** [comment]
Fix has gone out with v2.34.1. Closing this for now, don't hesitate to open a new ticket if you encounter any problems.
